recipe/meta.yaml

```diff
@@ -30,6 +31,8 @@ requirements:
     - liblapack
     - xdrfile
     - zlib
+  run:
+    - llvm-openmp  # [osx]
```

Comment:
You need this in host too.

Reply:
@isuruf thanks! It looks it is not sufficient however..